package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "applicationinsights",
   "license": "MIT",
-  "bugs": "https://github.com/Microsoft/AppInsights-node.js/issues",
+  "bugs": "https://github.com/Microsoft/ApplicationInsights-node.js/issues",
   "version": "0.12.5",
```

Comment:
definitely, thanks for the reminder. This will be 0.15 to align with the c#/browser JS release.